### PR TITLE
fix: Dashboard docs

### PR DESCRIPTION
**Pull request by notjustinshaw**

### CODE REVIEW

**Comment by notjustinshaw** [comment]
Closed #3 so now we have a single docs branch to work from.

File: components/chart.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```tsx App.tsx
+import { QuillProvider, Chart } from "@quillsql/react";
+
+function App() {
+	return (
+		<QuillProvider
+			organizationId={organizationId}
+			publicKey={API_KEY}
+			authToken={user.jwt}
```
Comment (shawnmagee1) [inline components/chart.mdx]: auth token doesn't exist

**Review by shawnmagee1 — CHANGES_REQUESTED** [review]

File: components/auth.mdx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Authentication"
+description: "Secure your data with token-based authentication"
```
Comment (shawnmagee1) [inline components/auth.mdx]: what is this

**Review by shawnmagee1 — CHANGES_REQUESTED** [review]

File: components/chart.mdx (hunk 1 above)
Comment (notjustinshaw) [reply components/chart.mdx]: Yeah that was old, removed this in the new version.

File: components/auth.mdx (hunk 2 above)
Comment (notjustinshaw) [reply components/auth.mdx]: Was going to explain the queryHeaders more here but ended up deleting this page. We discuss queryHeaders a little bit in the selfhost docs.

**Pull request merged**